lib/resources/orders.js

```diff
@@ -63,6 +63,7 @@ module.exports = function (api) {
         currency,
         receipt,
         method,
+        partial_payment: normalizeBoolean(partial_payment),
         payment_capture: normalizeBoolean(payment_capture),
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏`payment_capture` is not required now right in create order?

Reply:
@sidag95 for recurring payment order (emandate and register emandate) its required